From fa494f08354aad11f7c4a6aa8b012fd7f11569e0 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Thu, 27 Jan 2022 11:18:09 +0100
Subject: [PATCH 044/529] clk: stm32mp13: removal of LSE clock on RNG clock
 switch in RCC

RNG clock switch becomes 'reserved' on cut1.1

Change-Id: Ifafc3bb28e162b229ed1307ae9e6e4a7df9d4326
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/237216
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 drivers/clk/stm32/clk-stm32mp13.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/clk/stm32/clk-stm32mp13.c
+++ b/drivers/clk/stm32/clk-stm32mp13.c
@@ -581,7 +581,7 @@ static const char * const qspi_src[] = {
 };
 
 static const char * const rng1_src[] = {
-	"ck_csi", "pll4_r", "ck_lse", "ck_lsi"
+	"ck_csi", "pll4_r", "reserved", "ck_lsi"
 };
 
 static const char * const saes_src[] = {
